src/Tizen.TV.UIControls.Forms/TVESEnumerations.cs

```diff
@@ -0,0 +1,294 @@
+        /// Buffer already full
+        /// </summary>
+        Full,
+        /// <summary>
+        /// Submit succeeded
+        /// </summary>
+        Success 
+    }
+
+    /// <summary>
+    /// Enumerator for video mime type
+    /// </summary>
+    public enum VideoMimeType
+    {
+        /// <summary>
+        /// H.263
+        /// </summary>
+        H263,
+        /// <summary>
+        /// H.254
+        /// </summary>
+        H264,
+        /// <summary>
+        /// HEVC
+        /// </summary>
+        Hevc,
+        /// <summary>
+        /// MPEG-1
+        /// </summary>
+        Mpeg1,
+        /// <summary>
+        /// MPEG-2
+        /// </summary>
+        Mpeg2,
+        /// <summary>
+        /// MPEG-4
+        /// </summary>
+        Mpeg4,
+        /// <summary>
+        /// VP8
+        /// </summary>
+        Vp8,
+        /// <summary>
+        /// VP9
+        /// </summary>
+        Vp9,
+        /// <summary>
+        /// WMV3
+        /// </summary>
+        Wmv3
+    }
+
+    /// <summary>
+    /// Enumerator for audio mime type
+    /// </summary>
+    public enum AudioMimeType
+    {
+        /// <summary>
+        /// Unknown
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// AAC
+        /// </summary>
+        Aac,
+        /// <summary>
+        /// MP2
+        /// </summary>
+        Mp2,
+        /// <summary>
+        /// MP3
+        /// </summary>
+        Mp3,
+        /// <summary>
+        /// AC3
+        /// </summary>
+        Ac3,
+        /// <summary>
+        /// EAC3
+        /// </summary>
+        Eac3,
+        /// <summary>
+        /// VORBIS
+        /// </summary>
+        Vorbis,
+        /// <summary>
+        /// OPUS
+        /// </summary>
+        Opus,
+        /// <summary>
+        /// PCM_S16LE
+        /// </summary>
+        PcmS16le,
+        /// <summary>
+        /// PCM_S16BE
+        /// </summary>
+        PcmS16be,
+        /// <summary>
+        /// PCM_U16LE
+        /// </summary>
+        PcmU16le,
+        /// <summary>
+        /// PCM_U16BE
+        /// </summary>
+        PcmU16be,
+        /// <summary>
+        /// PCM_S24LE
+        /// </summary>
+        PcmS24le,
+        /// <summary>
+        /// PCM_S24BE
+        /// </summary>
+        PcmS24be,
+        /// <summary>
+        /// PCM_U24LE
+        /// </summary>
+        PcmU24le,
+        /// <summary>
+        /// PCM_U24BE
+        /// </summary>
+        PcmU24be,
+        /// <summary>
+        /// PCM_S32LE
+        /// </summary>
+        PcmS32le,
+        /// <summary>
+        /// PCM_S32BE
+        /// </summary>
+        PcmS32be,
+        /// <summary>
+        /// PCM_U32LE
+        /// </summary>
+        PcmU32le,
+        /// <summary>
+        /// PCM_U32BE
+        /// </summary>
+        PcmU32be
+    }
+
+    /// <summary>
+    /// Enumerator for error type from TVESPlayer
+    /// </summary>
+    public enum ErrorType

```

Comment:
It is too common naming
```suggestion
    public enum ESPlayerErrorType

```

Reply:
Fixed it. Thank you.